docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -12,7 +12,7 @@ This Onvif Camera Device Service is developed to control/communicate ONVIF-compl
 
 ## OpenAPI Spec
 The latest version 2.2.0 of the device service API specifications can be found
```

Comment:
It should be 2.3.0 right? And this version needs to be replaced in the rest of the doc too.

Reply:
`{{version}}` shows `main` at the moment

Reply:
I see still `2.2.0` in one place at least that needs to be replaced.